lib/rest_client/jogger/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module RestClient
   module Jogger
-    VERSION = '1.2.1'.freeze
+    VERSION = '1.2.2'.freeze
```

Comment:
No need for `.freeze` method here since `# frozen_string_literal: true` is here
```suggestion
    VERSION = '1.2.2'
```

Reply:
Nice catch, I'll amend and fix 👍